boards/BMS/Src/main.c

```diff
@@ -258,20 +260,24 @@ int main(void)
         Io_LTC6813CellVoltages_GetPackVoltage,
         Io_LTC6813CellVoltages_GetAverageCellVoltage);
 
+    ts = App_TractiveSystem_Create(
+        Io_Adc_GetAdc2Channel4Voltage,
```

Comment:
I think the function entered should be `Io_Adc_GetAdc1Channel3Voltage()` for acquiring the differential tractive system voltage

![image](https://user-images.githubusercontent.com/55927496/166121993-d7d2ceb8-9eb3-45de-ae50-8910125a29e7.png)